### PR TITLE
Feature/add delete

### PR DESCRIPTION
**Pull request by Yonasle**
-Add and Delete To do list 
-Saving and retrieving from Local Storage 
- Professional Readme file
- linter implemented
- webpack configured

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    showDeleteIcon: false, // Set showDeleteIcon to false initially
+    showEdit: false,
+  };
+  tasks.push(newTask);
+  saveTasksToLocalStorage();
+}
+
+export function deleteTask(index) {
+  tasks = tasks.filter((task) => task.index !== index);
+  /* eslint-disable no-use-before-define */
```
Comment (zieeco) [inline src/app.js]: - Hey,  please let's remove this linter disable and fix the linter, instead, move this function
```js
export function deleteTask(index) {
  tasks = tasks.filter((task) => task.index !== index);
  updateIndexes();
  saveTasksToLocalStorage();
}
``` 
immediately after the `updateIndexes()` function to fix the linter issue.

File: src/app.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  updateIndexes();
+  saveTasksToLocalStorage();
+}
+
+export function editTask(index, newDescription) {
+  tasks[index].description = newDescription;
+  saveTasksToLocalStorage();
+}
```
Comment (zieeco) [inline src/app.js:41]: - If a user tries to edit a task, it doesn't seem to work as expected. on pressing the keyboard, nothing seem to be working. please, double check this behaviour.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Yonasle,




Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- CRUD almost complete :heavy_check_mark: 

# Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
_Check the comments under the review._

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @Yonasle, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**